shell/config/router/routes.js

```diff
@@ -379,6 +379,10 @@ export default [
         path:      '/c/:cluster/settings/performance',
         component: () => interopDefault(import('@shell/pages/c/_cluster/settings/performance.vue')),
         name:      'c-cluster-settings-performance'
+      }, {
+        path:      '/c/:cluster/settings/fleet',
```

Comment:
if UX wants to change the label on the Settings page, I would also adjust the path